lib/constants.js

```diff
@@ -94,6 +94,31 @@ module.exports = {
         "claim": 600
     },
 
+    RANGE_ATTACK: 1,
+    RANGE_ATTACK_CONTROLLER: 1,
+    RANGE_BOOST_CREEP: 1,
+    RANGE_BUILD: 3,
+    RANGE_CLAIM_CONTROLLER: 1,
+    RANGE_DISMANTLE: 1,
+    RANGE_DROP: 1,
```

Comment:
It's possible that I just mindlessly added RANGE_ constants for all of the actions, it obviously doesn't make any sense for drop.
I'll update the PR later on.